examples/torque_driven_ocp/Trampo_Quaternions.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import biorbd
+
+from biorbd_optim import (
+    OptimalControlProgram,
+    ProblemType,
+    Objective,
+    Bounds,
+    QAndQDotBounds,
+    InitialConditions,
```

Comment:
I think a proper test would be required (of a minimal example so it is not too long) as well